src/main/scala/cromwell/binding/Scope.scala

```diff
@@ -1,14 +1,168 @@
 package cromwell.binding
 
+import cromwell.binding.AstTools.AstNodeName
+import cromwell.parser.WdlParser.{AstList, Ast, AstNode}
+import scala.collection.JavaConverters._
+
+
+object Scope {
+
+  /**
+   * Utility class to generate a ScopeTree from an Ast.
+   */
+  private class ScopeTreeGenerator {
+
+    /* Keep track of the # of created Scatter nodes. */
+    var scatterCount: Int = 0
```

Comment:
Right, it's not necessary but it's used to keep counters only for classes that will actually use it.
Removing it would make the function even shorter though.